apps/docs/pages/guides/platform/oauth-apps/oauth-scopes.mdx

```diff
@@ -0,0 +1,43 @@
+| `Edge Functions` | `Write` | Create, update, delete an edge function                                                                                                                                                                                                                                                   |
+| `Environment`    | `Read`  | Retrieve branches in a project                                                                                                                                                                                                                                                            |
+| `Environment`    | `Write` | Create, update, or delete a branch                                                                                                                                                                                                                                                        |
+| `Organizations`  | `Read`  | Retrieve an organization's metadata<br/>Retrieve all members in an organization                                                                                                                                                                                                           |
+| `Organizations`  | `Write` | Create an organization                                                                                                                                                                                                                                                                    |
+| `Projects`       | `Read`  | Retrieve a project's metadata<br/>Check if a project's database is eligible for upgrade<br/>Retrieve a project's network restrictions<br/>Retrieve a project's network bans                                                                                                               |
+| `Projects`       | `Write` | Create a project<br/>Upgrade a project's database<br/>Remove a project's network bans<br/>Update a project's network restrictions                                                                                                                                                         |
+| `Rest`           | `Read`  | Retrieve a project's postgREST configuration                                                                                                                                                                                                                                              |
+| `Rest`           | `Write` | Update a project's postgREST configuration                                                                                                                                                                                                                                                |
```

Comment:
This should be `PostgREST`, right?

Reply:
not really, we decided to call it "Rest" because that's what we call it at the product level: 
<img width="268" alt="image" src="https://github.com/supabase/supabase/assets/28647601/bfa2674e-70bd-4a4b-aa66-040c99579258">